easybuild/easyconfigs/m/MoreRONN/MoreRONN-4.9-foss-2019b.eb

```diff
@@ -0,0 +1,37 @@
+# EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Jordi Blasco <jordi.blasco@hpcnow.com>
+# HPCNow! - www.hpcnow.com
+
+easyblock = 'ConfigureMake'
+
+name = 'MoreRONN'
+version = "4.9"
+
+homepage = 'https://github.com/varun-ramraj/MoreRONN'
+description = """MoreRONN is the spiritual successor of RONN and is useful for surveying disorder
+ in proteins as well as designing expressible constructs for X-ray crystallography."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'pic': True, 'usempi': False}
```

Comment:
with no mpi, and seemingly no BLAS/LAPACK dependencies, seems like this could be moved down to GCC level

Reply:
Created new PR using GCC toolchain #10255. This PR can be closed